cmd/aries-agent-mobile/pkg/api/storage.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+// StoreIterator is the iterator for the latest snapshot of the underlying store.
+type StoreIterator interface {
+	// Next moves the iterator to the next key/value pair.
+	// It returns false if the iterator is exhausted.
+	Next() bool
+
+	// Free releases associated resources. Release should always success
+	// and can be called multiple times without causing error.
+	Free()
```

Comment:
are we then going to remove the duplicate interfaces in agent-sdk?

Reply:
I don't think it will work. But I can try. https://github.com/trustbloc/agent-sdk/pull/49#discussion_r520005584